app/views/admin/editions/locked.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_title @edition.title, @edition.format_name %>
+<% page_class "edition-show" %>
```

Comment:
Depending on how much CSS is actually shared with this, it might make sense for it to be a separate scss file just for locked.

Reply:
We were using `.edition-view-edit-buttons` from the edition-show CSS file but it doesn't seem to have any impact if it's removed so will create a new file for locked CSS.